### DISCUSSION TITLE
ubuntu server 22 install

### DISCUSSION THREAD

**Discussion by bserret** [Ylianst/MeshCentral#5970] [General]
I am wanting to install on ubuntu server 22 - should I follow the docs section for secure server (just after the ubuntu 18 install section in docs) or can someone suggest a better approach? the server will be behind reverse proxy using nginx proxy manager which will handle certs.

Thanks!

**Comment by bserret** [comment]
when I attempt that install "sudo npm install meshcentral" generates a number of warning errors (EBADENGINE Unsupported engine) for various modules and the install fails.

**Reply by bserret** [reply]
I believe I have it working - seems the primary fix was to upgrade nodejs to v20.

**Reply by si458** [reply]
YES you MUST have at least node 18 or above now,
DONT install with `apt install nodejs` this will install an OLD nodejs version!

instead use something like nvm (which nodejs now recommends)
https://nodejs.org/en/download/package-manager
OR
nodesource which install the packages via `apt`
https://github.com/nodesource/distributions?tab=readme-ov-file#installation-instructions
